Kiosk/Bid Fulfillment/BidderNetworkModel.swift

```diff
@@ -110,7 +110,7 @@ private extension BidderNetworkModel {
             .filterSuccessfulStatusCodes()
             .map(void)
             .doOnCompleted { [weak self] in
-                // Adding the credit card succeeded, so we shoudl clear the newUser.creditCardToken so that we don't
+                // Adding the credit card succeeded, so we should clear the newUser.creditCardToken so that we don't
                 // inadvertently try to re-add their card token if they need to increase their bid.
 
                 self?.bidDetails.newUser.creditCardToken.value = nil
```

Comment:
I think this is one area where bugs tend to creep in -- we have mixes of FRP and of shared mutable state, which I think has the potential of dropping us to the lowest common denominator of reliability.

But not sure that's actionable right now, other than as a note to ourselves.